packages/clerk-js/package.json

```diff
@@ -113,7 +113,7 @@
     "files": [
       {
         "path": "./dist/clerk.js",
-        "maxSize": "162kB"
+        "maxSize": "164kB"
```

Comment:
Not sure what the policy on bundle watch here is, but it went up on the latest rebase